20230810_foregrounds_cmb/common.toml

```diff
@@ -0,0 +1,9 @@
+lmax_over_nside = 2.5
+healpix = true
+car = false
+output_folder = "output/{tag}"
+output_filename_template= "litebird_{tag}_uKCMB_{telescope}-{band}_nside{nside}.fits"
```

Comment:
@giuspugl I'd prefer to keep same naming convention used for S4, but if you prefer otherwise please let me know.